metaflow/plugins/argo/argo_workflows.py

```diff
@@ -179,10 +180,14 @@ def _cron(self):
             return " ".join(schedule.schedule.split()[:5])
         return None
 
+    def _timezone(self):
```

Comment:
small nit - can we have `_cron(...)` return a tuple that includes the timezone as well. that will negate the need for creating this function.

Reply:
Done.